### PR TITLE
Bump to latest conda

### PR DESCRIPTION
**Pull request by agjohnson**
Conda 4.4.10

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ LABEL version="latest"
 ENV DEBIAN_FRONTEND noninteractive
 ENV APPDIR /app
 ENV LANG C.UTF-8
-ENV CONDA_VERSION 4.3.31
+ENV CONDA_VERSION 4.4.10
```
Comment (humitos) [inline Dockerfile]: `4.4.11` is the latest now https://github.com/conda/conda/blob/master/CHANGELOG.md#4411-2018-02-23

Comment (agjohnson) [reply Dockerfile]: 4.4.10 is the latest miniconda release though

**Comment by agjohnson** [comment]
Yeah, i think we just need to run latest. If there are any bad releases, we'll get notified of bugfix releases to miniconda

**Comment by jakirkham** [comment]
IDK if you saw this, but how one [activates `conda` has changed in 4.4]( https://github.com/conda/conda/blob/master/CHANGELOG.md#440-2017-12-20 ).

**Comment by humitos** [comment]
Thanks for let us know about this.

I opened an issue at #63

@jakirkham if you want to contribute with a PR, you are more than welcome ;)

**Comment by jakirkham** [comment]
Kind of swamped ATM, but happy to look over a PR if you need another set of eyes.

**Pull request merged**